requirements-common.txt

```diff
@@ -5,7 +5,7 @@ sentencepiece  # Required for LLaMA tokenizer.
 numpy
 requests
 py-cpuinfo
-transformers >= 4.40.0  # Required for StarCoder2 & Llava, Llama 3.
+transformers >= 4.41.0  # Required for StarCoder2 & Llava, Llama 3.
```

Comment:
Because we used `AutoConfig.from_pretrained` to create model config from gguf checkpoint in `transformers_utils/config.py`. And this feature is introduced in `transformers >= 4.41.0`.